subsys/bluetooth/rpc/common/serialize.c

```diff
@@ -324,6 +324,51 @@ int32_t ser_decode_int(CborValue *value)
 	return 0;
 }
 
+void ser_decode_str(CborValue *value, char *buffer, size_t buffer_size)
```

Comment:
Most of the `ser_` decoding functions that process buffers return buffer pointer or NULL when they fail, this allows quickly to find out whether decoding went fine. Can we do that here too?

Reply:
Sure will be done under #7655